src/main/java/org/jenkinsci/plugins/oic/OicAvatarResolver.java

```diff
@@ -0,0 +1,21 @@
+package org.jenkinsci.plugins.oic;
+
+import hudson.Extension;
+import hudson.model.User;
+import hudson.tasks.UserAvatarResolver;
+
+@Extension(ordinal = -1)
```

Comment:
not needed, I may have forgot to remove this from the Azure one but I bumped the Avatar  plugin priority up so the default is fine
```suggestion
@Extension
```